sdk/translation/azure-ai-translation-text/README.md

```diff
@@ -37,10 +37,12 @@ pip install azure-ai-translation-text
 
 #### Create a Translator service resource
 
-You can create Translator resource following [Create a Translator resource][translator_resource_create].
+You can create Translator resource following [Create a Translator resource][translator_resource_create].  For general information about authentication with a cognitive services resource please refer to the documentation [here][cognitive_auth].
 
-### Authenticate the client
+### Endpoint
+The values of the `endpoint` variable can be retrieved from environment variables, configuration settings, or any other secure approach that works for your application.  Four different types of authentication are available for different types of endpoints: global, regional, token and Entra (AAD).  For details about each including curl examples refer to the documentation available [here][authentication_details].  These samples demonstrate API Key and Entra (AAD) authentication.
```

Comment:
> different types of endpoints: global, regional, token and Entra 

I am not sure I understand what endpoint is "token" and what endpoint is "Entra (AAD)". Are you mixing here endpoint types and authentication methods?

Reply:
Corrected.